examples/bootstrap.php

```diff
@@ -3,7 +3,7 @@
 /**
  * Bootstrap the library
  */
-require_once __DIR__ . '/../vendor/autoload.php';
+require_once __DIR__ . '/../src/OAuth/bootstrap.php';
```

Comment:
The `vendor` folder is created by [Composer](https://getcomposer.org) which is the package manager for PHP. You should run:

```bash
composer install
```